test/utils.ts

```
@@ -0,0 +1,17 @@
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const fixtures = path.join(__dirname, 'fixtures');
+
+export function getFilepath(name: string) {
+  return path.join(fixtures, name);
+}
+
+export function escape(str: string) {
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)